plugin/goto.py

```diff
@@ -59,9 +58,7 @@ def handle_response(self, response: Any, side_by_side: bool) -> None:
         if window is None:
             return
         if response:
-            # Save to jump back history.
-            get_jump_history_for_view(view).push_selection(view)
-            # TODO: DocumentLink support.
+            self.view.run_command("add_jump_record", {"selection": [(r.a, r.b) for r in self.view.sel()]})
```

Comment:
Id null-check self `view.sel()` maybe?

Also, there is `view` in scope (short than `self.view`).